src/stash/protos/objects.proto

```diff
@@ -238,16 +238,6 @@ enum CatalogItemType {
     CATALOG_ITEM_TYPE_CONNECTION = 10;
 }
 
-message CatalogItem {
-    message V1 {
-        string create_sql = 1;
-    }
-
-    oneof value {
-        V1 v1 = 1;
-    }
-}
-
```

Comment:
What do you think about keeping the `CatalogItem` message? I feel like one day we might want to persist more than just the `create_sql`, and keeping the message might make that easier? Not sure though

Reply:
Updated the PR to revert the stash changes.